### ISSUE TITLE
about pred_pose

### ISSUE THREAD

**Issue by matlab2017** [haofanwang/CLIFF#10]
I'm a beginner in human reconstruction. I want to ask  why pred_rotmat is 24×3×3 obtained by follows:

with torch.no_grad():
            pred_rotmat, pred_betas, pred_cam_crop = cliff_model(norm_img, bbox_info)

What is the difference between it and the pose parameters θ in the smpl model. Can I interpret it this way: pred_rotmat (24×3×3) can transfer to pred_pose (1×72) according following, and  pred_pose (1×72) is equal to pose parameters θ ??

            if args.pose_format == "aa":
                rot_pad = torch.tensor([0, 0, 1], dtype=torch.float32, device=device).view(1, 3, 1)
                rot_pad = rot_pad.expand(pred_rotmat.shape[0] * 24, -1, -1)
                rotmat = torch.cat((pred_rotmat.view(-1, 3, 3), rot_pad), dim=-1)
                pred_pose = tgm.rotation_matrix_to_angle_axis(rotmat).contiguous().view(-1, 72)  # N*72
